### PR TITLE
Adding gauripowale as a dcos owner.

### PR DESCRIPTION
**Pull request by gauripowale**
Adding gauripowale from the Testing team as a dcos owner.

### CODE REVIEW

**Comment by gauripowale** [comment]
@mesosphere-mergebot label ready for review

File: owners.json [hunk 1]
```diff
@@ -12,5 +12,6 @@
   "orlandohohmeier": {"slack": "orlando", "jira": "orlando"},
   "orsenthil" : {"slack": "skumaran", "jira": "skumaran"},
   "rukletsov": {"slack": "alexr", "jira": "alexr"},
-  "urbanserj": {"slack": "urbanserj", "jira": "sergeyurbanovich"}
+  "urbanserj": {"slack": "urbanserj", "jira": "sergeyurbanovich"},
+  "gauripowale": {"slack": "gpowale", "jira": "gauripowale"}
```
Comment (orsenthil) [inline owners.json]: Hi @gauripowale  - we've tried to maintain this in sorted order. Please sort this addition, and I will approve again.

**Comment by gauripowale** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Pull request merged**